README.md

```diff
@@ -1,2 +1,80 @@
 # asset-pipe-css-writer
 
+A module that takes any number of css file entry points and packages them together with meta data before providing them as a readable stream.
+
+## Overview
+
+Given any number of css file paths, for each file path, this module will:
+1. fetch the file at the path
+2. fetch a name and version from the nearest package.json to the file
+3. bundle the css found in the file (resolving any @import statements and inlining them)
+4. put all this together in an object (See Output data format below)
+
+The module provides a readable stream of the resulting objects.
+
+### Output data format
+
+```js
+{
+    // Unique id for entry. Created by hashing together name, version and file
```

Comment:
not the content of the file?

Reply:
hmm, yea I guess I was operating on the assumption that id,name,version,file would be enough since they should be A. unique and B. would change if the package was updated so to include content as well would perhaps handle the case where if somehow the package was updated without updating the version. 
I'll change it to include content as well

Reply:
The use case is e.g. linking some files together locally, I suppose.

The js writer only looks at content. Should those implementations be aligned?

https://github.com/asset-pipe/asset-pipe-js-writer/blob/dac298c19b285a8ae2a64d76a8c894bdc198d773/lib/writer.js#L20